aspnetcore/security/data-protection/configuration/scaling/includes/scaling7.md

```diff
@@ -0,0 +1,8 @@
+
+:::moniker range="< aspnetcore-8.0"
```

Comment:
Out of curiosity, why add docs to 7.0, which is out of support, but 6.0, which is not?